### PR TITLE
lesson 6

### PR DESCRIPTION
**Pull request by lolitamalinovskaya**
added block 1 according to design. I'm having problems with :focus functionality and don't know how to make a calendar with 2 dates

### CODE REVIEW

File: Style.css [hunk 1]
```diff
@@ -18,6 +18,232 @@ body {
...
+    content: "";
+    height: 4px;
+    width: 110px;
+    background-color: #F5BD41;
+    display: block;
+    position: relative;
+    top: 10px;
+    left: calc((100% - (110px)) / 2);
+    margin-top: -4px;
+}
```
Comment (romos21) [inline Style.css]: @lolitamalinovskaya properties for classes are simple: you can set it like this:  .stays::after, .attractions:hover::after {}
also I think it should works only if hover on for 2 elements - not only for attractions

File: Style.css [hunk 2]
```diff
@@ -18,6 +18,232 @@ body {
...
+    box-sizing: border-box;
+    width: 1440px;
+    height: 810px;
+}
+
+.top_section_main_general_label {
+    display: flex;
+    box-sizing: border-box;
+    height: 145px;
+    width: 760px; /*на самом деле 640*/
```
Comment (romos21) [inline Style.css:125]: XD

File: Style.css [hunk 3]
```diff
@@ -18,6 +18,232 @@ body {
...
+    height: 65px;
+    width: 1230px;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: auto;
+    align-items: baseline;
+    background-color: #FFFFFF;
+    border-radius: 8px;
+
+}
```
Comment (romos21) [inline Style.css]: @lolitamalinovskaya set one common class here (all properties are equal) - the difference only in bg-color

File: Style.css [hunk 4]
```diff
@@ -18,6 +18,232 @@ body {
...
+
+#check-in, #check-out {
+    font: 400 18px Roboto;
+    border: none;
+    border-radius: 8px;
+    outline: none;
+}
+
+#top_section_main_input_for_check-in_check-out { /*как настроить плэйсхолдер на текст а не на дату?*/
```
Comment (romos21) [inline Style.css:183]: I hope that inline property should works for all types of input... and of course for buttons

**Comment by romos21** [comment]
Also, I think that there too many width and height properties for blocks

File: Style.css (hunk 1 above)
Comment (lolitamalinovskaya) [reply Style.css]: Dasha told us to stay highlighted as we are now on the stays tab. And I cannot combine these two properties as the lines have different lengths and margins

File: Style.css (hunk 3 above)
Comment (lolitamalinovskaya) [reply Style.css]: I removed the identical class

**Comment by lolitamalinovskaya** [comment]
I solve this problem, in the next homework this will not be

**Pull request merged**